app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java

```diff
@@ -82,7 +84,7 @@ public static void clearPin() {
 
     public static void initialize() {
         if (!m_communicationStarted) {
-            UserError.Log.i(TAG, "initialize!");
+            Log.i(TAG, "initialize!");
             Home.setPreferencesInt("bridge_battery", 0); //force battery to no-value before first reading
```

Comment:
The mechanism that xDrip uses is Log. It's advantage is that it is kept in the DB for later processing. Users can also see it without a computer, and send it for the developer to get help.
It will later call the UserError function.

Reply:
@gregorybel tzachi imports `UserError.Log` above so that `Log.i` is `UserError.Log.i` this is preferable shorthand allowing `Log` to be replaced with xDrip's `UserError.Log` but it only works when the class is in the same package.

Reply:
Il!

Reply:
Ok!